spec/ddtrace/contrib/rails/redis_cache_spec.rb

```diff
@@ -29,8 +29,8 @@
   before { app }
 
   before do
-    Datadog.configure { |c| c.use :redis }
-    Datadog.configure(client_from_driver(driver), tracer_options)
+    Datadog.configure { |c| c.use :redis, tracer: tracer }
+    Datadog.configure(client_from_driver(driver))
```

Comment:
Given it is not possible to pass `tracer:` to the Pin anymore (which happens through `Datadog.configure(client)` here), we have to move the tracer configuration to the integration level.

This explicit `tracer:` configuration at integration level is also going away in a future PR, but not on the scope of this one.